tests/test_well/test_well.py

```diff
@@ -455,7 +525,7 @@ def test_mask_shoulderbeds_use_tvd_md(loadwell3):
     assert "no mdlogname attribute present" in str(verr)
 
     usewell.geometrics()  # to create Q_MDEPTH as mdlogname
-
+    # verify that this tests geometrics (and could it be done more explicitly?)
```

Comment:
I think it might be a good idea to settle the matter and not add the comment. I don't really understand what is suggested so I think just leave it as is :)

Reply:
Haha, good point, it stems from when I inherited the PR, and forgot to change it. Will fix it!